components/gitlab/new-milestone.js

```diff
@@ -0,0 +1,91 @@
+const gitlab = require("https://github.com/PipedreamHQ/pipedream/components/gitlab/gitlab.app.js");
+
+module.exports = {
+  name: "New Milestone (Instant)",
+  description: "Triggers when a new milestone is created in a project",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    gitlab,
+    projectId: { propDefinition: [gitlab, "projectId"] },
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60,
```

Comment:
Since free users have limited quota, I like to default sources to `15 * 60` `intervalSeconds`. Then since the user can change the default on component creation, they can always set a lower time.

Reply:
Done